.drone.jsonnet

```diff
@@ -20,7 +26,7 @@ local pipeline(arch, darch, tags) = {
           from_secret: 'docker_pass',
         },
         repo: 'clandmeter/docker-abuild',
-        tags: '%s-%s' % [ std.strReplace(tag,'v',''), arch],
+        tags: '%s-%s' % [std.strReplace(tag, 'v', ''), arch],
         dockerfile: 'Dockerfiles/%s/%s/Dockerfile' % [tag, arch],
```

Comment:
Should we add `Dockerfiles` to `.gitignore` now?

Reply:
Yes i think so